de.seronet_projekt.mixedport.ros.feature/feature.xml

```diff
@@ -100,49 +100,49 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
          id="de.seronet_projekt.ros.interfaces.pool.xtext.ide"
          download-size="0"
          install-size="0"
-         version="1.0.0.qualifier"
+         version="0.0.0"
          unpack="false"/>
 
    <plugin
          id="de.seronet_projekt.ros.interfaces.pool.xtext"
          download-size="0"
          install-size="0"
-         version="1.0.0.qualifier"
+         version="0.0.0"
          unpack="false"/>
 
    <plugin
          id="de.seronet_projekt.ros.interfaces.pool.xtext.ui"
          download-size="0"
          install-size="0"
-         version="1.0.0.qualifier"
+         version="0.0.0"
          unpack="false"/>
 
    <plugin
          id="de.seronet_projekt.xtend.ROS.generator"
          download-size="0"
          install-size="0"
-         version="1.0.0.qualifier"
+         version="0.0.0"
```

Comment:
It is better to remove the exact versions from the feature and let the feature automatically take the newest available plugin version. This will prevent forgotten updates in the future.